hack/make/containerutility

```diff
@@ -8,6 +8,13 @@ CONTAINER_UTILITY_COMMIT=e004a1415a433447369e315b9d7df357102be0d2 # v0.9.0
 	cd "$GOPATH/src/github.com/docker/windows-container-utility"
 	git checkout -q "$CONTAINER_UTILITY_COMMIT"
 
+	# TODO remove this temporary fix once https://github.com/docker/windows-container-utility/pull/2 is merged
+	sed -i \
```

Comment:
nit: Would it make more sense to create a patch for this file and then apply the patch instead? This way if something changes upstream the patch might start failing instead of sed just silently replacing something it's not supposed to, or not changing something it doesn't find.

Reply:
It'll likely be merged soon; some discussion was still happening, but this should go away soon

Reply:
Merged the fix in the windows-container-utility repo, and opened a PR to revert this temporary fix: https://github.com/moby/moby/pull/40526